.circleci/config.yml

```diff
@@ -492,14 +492,14 @@ jobs:
             npm install -g --silent gh-pages@2.0.1
             gh-pages --dotfiles --message "[skip ci] Updates" --dist build/docs
 
-  Lint Python with flake8 and black:
+  Lint Python with flake8 and black and mypy:
```

Comment:
This becomes unhandy. Should we just name it `Lint Python` and not bother to change it with every new tool we add?